configuration/concepts/lab_tests.csv

```diff
@@ -1,110 +1,110 @@
-ef5d7595-87f6-4814-95e0-41533a517783,,,Others,Otros,Otros,Others,,Sample,N/A,,,Miscellaneous;Other culture,,,,,,,,
-8160a011-3f10-11e4-adec-0800271c1b75,,,Lab Samples,Muestras de laboratorio,Muestras de laboratorio,Laboratory,,Misc,N/A,,,Blood; Urine; Swab; Stool; Others,,,,,,,,
-b78fbee4-3c34-4a27-973c-7673822c1c3e,,,Other Orderables,Otra,Otra,Other Orderables,,Misc,N/A,,,Cardiology Orders,,,,,,,,
-a3a4db93-ba92-492d-aa3a-f19476bd1d73,,,All_Tests_and_Panels,All_Tests_and_Panels,All_Tests_and_Panels,All_Tests_and_Panels,,Misc,N/A,,,Hematocrit;Red Blood Cells;White Blood Cells;Platelets;MCV;MCH;MCHC;RDW;Reticulocytes;Neutrophil;Lymphocyte; Monocyte;Basophil;Eosinophil;Creatinine;BUN;Sodium;Potassium;Calcium Test;Bicarbonate;Chloride;Phosphate;Serum magnesium;Uric acid; Total Protein; Total Bilirubin;Direct Bilirubin;albumin;ALT;AST;ALP;Total Cholesterol;HDL;LDL;Triglycerides;PT;aPTT;INR;C-Reactive Protein;Erythrocyte Sedimentation Rate;TSH;T3;T4;Amylase;Lipase;Rheumatoid Factor;Antistreptolysin O;Prolactin;Sex Hormones;Urine culture;PH;Specific gravity;Glucose;Ketones;Nitrites;Leukocyte esterase;Bilirubin;Blood in urine;Protein;Rapid HIV;Fasting blood glucose;Random blood glucose;HbA1C;Hemoglobin;Rapid malaria;Urine pregnancy test;EKG;Blood culture;Stool culture;Wound culture;Other culture;Fecal occult;Bacteria;Clue cells;RBCs;Ferning;Whiff Test;Urinalysis;CBC;Thyroid Functional Test;Renal Functional Test;Liver Functional Test;Lipid Profile;Yeast;Bacterial vaginosis;Trichomonas;Vaginal Candidiasis;ab6588f6-87ee-45ca-a6c5-dc1337f83469;Chlamydia;Miscellaneous,,,,,,,,
-b3dac224-fdea-11e4-b248-005056820298,,,All Orderables,Todos los pedidos,Todos los pedidos,All Orderables,,ConvSet,N/A,,,Lab Samples;Other Orderables,,,,,,,,
\ No newline at end of file
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:es,Short name:es,Short name:en,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high
+718b4589-2a11-4355-b8dc-aa668a93e098,,,Negative,Negativo,Negativo,Negative,,Misc,N/A,,,,,,,,,
+7acfafa4-f19b-485e-97a7-c9e002dbe37a,,,Positive,Positivo,Positivo,Positive,,Misc,N/A,,,,,,,,,
+0a5d3d0a-eaad-4770-9d54-ae949a7e8a87,,,Normal,Normal,Normal,Normal,,Misc,N/A,,,,,,,,,
+790f812a-e827-4766-b6bd-ef51c534882f,,,Trace,Rastro,Rastro,Trace,,Misc,N/A,,,,,,,,,
+a6f85472-1bf5-4a05-bb38-7769ef3c6350,,,Small,Pequeña,Pequeña,Small,,Misc,N/A,,,,,,,,,
+3364e45c-e10d-4697-bb44-c26ff29a5e3a,,,Moderate,Moderada,Moderada,Moderate,,Misc,N/A,,,,,,,,,
+a0be77d9-3134-4877-b18a-f315f9c681f8,,,Large,Grande,Grande,Large,,Misc,N/A,,,,,,,,,
+97bc3471-a39c-44ce-8d01-f241e1418e41,,,Hematocrit,Hematocrito,Hematocrito,Hematocrit,,LabTest,Numeric,,,,Yes,%,39,52,,
+796a4dd1-a729-4d27-bea2-aa07496f303f,,,White Blood Cells,Células blancas de la sangre,Células blancas de la sangre,White Blood Cells,,LabTest,Numeric,,,,Yes,X10 *9L,5,10,,

```

Comment:
Let's change the unit of '**X10 *9L**' to '**x10^9/L**'
Let's do this for other tests as well

Reply:
Done